docs/build/guides/conversions/address-conversions.mdx

```diff
@@ -113,3 +101,28 @@ sc_address_xdr = address.to_xdr_sc_address()
 ```
 
 </CodeExample>
+
+## Conversions in Smart Contracts
+
+Smart contracts don't need to explicitly interact with the XDR types, as all the smart contract data types are automatically converted to XDR by the smart contract runtime. Every contract type, including `Address`, can be serialized to XDR bytes. This conversion is useful, for example, for performing hashing in smart contracts. It is also possible to convert the serialized XDR bytes back to contract types, which can be useful in some narrow use cases, such as custom authentication schemes.
+
+Note, that XDR conversions are and advanced feature and are not necessary for most of the Stellar smart contracts.
```

Comment:
I was under the impression that XDR serialization required `std` :thinking: is that not true?

Reply:
Nope, we have host functions for that. TBH I'm not sure if this section is useful, I just left it for the sake of consistency (because address<->strkey conversions are unique and make sense as a separate section). But maybe it's worth removing it and document the XDR conversions generically elsewhere...

Reply:
Gotcha. I do think it's useful, though - I don't think it's covered well elsewhere (could be wrong, in which case a cross reference is better) -- more is better either way.

Reply:
Yeah, I'm not saying this information is not useful at all, I'm rather questioning whether this belongs to a separate (not address-focused section). But for now it's probably better than nothing.